recipes/pygenomeviz/meta.yaml

```diff
@@ -18,19 +24,23 @@ requirements:
   host:
     - pip
     - poetry
-    - python >=3.7,<4.0
+    - python >=3.8,<4.0
```

Comment:
I strongly recommend removing the `<4.0`: https://iscinumpy.dev/post/bound-version-constraints/

Reply:
Thank you for presenting an interesting article on dependency version constraints. 
It was very helpful to learn about the side effects of version capping, which I had not been particularly aware of before.

Reply:
Thanks for making the changes! It is a very long article, but it was convincing when I read it.